### ISSUE TITLE
Create a nice HTML file to serve back when discord accounts are linked

### ISSUE THREAD

**Issue by zkldi** [TeamNewGuys/tachi-bot#24]
At the moment, we just return a string instead of a nice HTML file.

We could make something pretty and display that as the "congratulations your accounts are linked" thing?

Wouldn't need any sort of UI library, just a single HTML file will do.

**Comment by TheodosiouTh** [comment]
@zkldi  I would love to help! Can you assign me this issue?

**Comment by zkldi** [comment]
Sounds good! The page just needs to say something like "Your accounts are now linked!".

I can't provide much in terms of styling but the normal Tachi website looks like this:
![image](https://user-images.githubusercontent.com/20380519/132125541-e210eb3a-f5a0-49cc-beb2-74e9b83233a3.png)

Colour scheme wise, thats:
```scss
$tachi-primary: #e61c6e;
$tachi-primary-hover: #de6589;
$tachi-background: #131313;
$tachi-lightground: #2b292b;
```

Font is [poppins](https://fonts.google.com/specimen/Poppins). Thank you!

**Comment by TheodosiouTh** [comment]
I added a pull request. Feel free to check it out: #27

**Comment by zkldi** [comment]
Fixed with #27
